SequencePhenotype.java

```diff
@@ -1,50 +1,107 @@
-import java.util.*;
+import java.lang.*;
+
+/**
+ * <b>SequencePhenotype</b> represents a phenotype.
+ * SequencePhenotypes are identified by their data contents.
+ * SequencePhenotype permits all data, except for <b>""</b> and <b>null</b>.
+ *
+ * @author Thien Tran
+ */
 public class SequencePhenotype implements Phenotype {
 
-    // constants
-    @SuppressWarnings("unchecked")
-    public final String[] NUCLEOBASES = new String[]{"A", "C", "G", "T"};
+    /**
+     * The valid letters that make up the sequence of this SequencePhenotype
+     */
+    public final String[] NUCLEOTIDES = new String[]{"A", "C", "G", "T"};
```

Comment:
Maybe we should think about renaming this something like `ALPHABET` or `AMINO_ACIDS` if we're going to be leaving the nucleotide representation in the future (idk if we'll keep it around and have an option for either DNA or AAs). But doesn't need to be done until we get ready to cross that bridge.